docs/usage/delete-restore.md

```diff
@@ -0,0 +1,63 @@
+# Delete and restore knowledge
+
+Knowledge can be deleted from OpenCTI either in an overview of an object or using [background tasks](background-tasks.md)
+When we delete an object, we also delete all its relationships and references to other objects. 
+
+The deletion event is written to the [stream](../reference/streaming.md), to trigger automated [playbooks](./automation.md) or synchronize another platform.
+
+Prior to 6.1 version, there was no way to undo this operation and the knowledge was lost permanently.
+Since OpenCTI 6.1, we now keep record of the deleted objects for a given period of a time, allowing to restore them on demand. This does not impact the stream events or other side effect of the deletion: the object is still _deleted_.
```

Comment:
```suggestion
Since OpenCTI 6.1, a record of all the deleted objects is kept for a given period of time, allowing to restore them on demand. This does not impact the stream events or other side effect of the deletion: the object is still _deleted_.
```